### ISSUE TITLE
Official release in Amazon Appstore is not obvious / confusing

### ISSUE THREAD

**Issue by foosmate** [home-assistant/android#2518]
I found the Home Assistant app in the [Amazon Appstore](https://www.amazon.com/-/de/dp/B09RP3DM5T/ref=sr_1_13?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=2TBE78RX1XXJL&keywords=home+assistant+app&qid=1652277806&sprefix=home+assistant+app%2Caps%2C132&sr=8-13) (more or less by accident), but the app is not even listed on the HA [Companion Website](https://companion.home-assistant.io/) . 

Or is the app not official?

In order to *not* scare people (SCAM etc.) I would highly recommend having the app listed on the Companion site as well.

Since it makes things much easier to install on a Fire HD 10 (especially for people who don't want to tinker around) it would be great to spread the information.

**Comment by dshokouhi** [comment]
This is actually a soft launch until the app is available on windows 11 devices. We will make an announcement when that time comes.

**Comment by foosmate** [comment]
Is the app in the Amazon Appstore being maintained / updated regurlarly?

Current version is 2022.2.0 ...

**Comment by dshokouhi** [comment]
We haven't setup any type of automation yet to keep it up to date, I think we plan on doing that once were ready to announce things. The version of the app submitted is the Minimal version, you are free to grab the Minimal APK off the GitHub releases tab in the meantime.

**Comment by nareddyt** [comment]
> This is actually a soft launch until the app is available on windows 11 devices. We will make an announcement when that time comes.

What is the status of this? I tried finding the app in the Amazon Appstore on my windows 11 device, but it does not show up.

Does the app need to be re-submitted using https://developer.amazon.com/apps-and-games/appstore-on-windows-11?

**Comment by dshokouhi** [comment]
> What is the status of this?

no updates as of yet, still waiting on Amazon and Microsoft

> Does the app need to be re-submitted using https://developer.amazon.com/apps-and-games/appstore-on-windows-11?

we are not part of the closed beta

`Amazon Appstore on Windows 11 is currently available to a selection of apps as part of a closed beta. `
